modules/ding_provider/ding_provider.module

```diff
@@ -605,6 +598,85 @@ function ding_provider_get_pseudo_entity($ding_entity_id) {
   return $ting_entity;
 }
 
+/**
+ * Use this to try and create pseudo entities when materials are not searchable
```

Comment:
Every function should be described with a one-line comment. Further extended explanation can follow after a black line.